serious_django_permissions/groups.py

```diff
@@ -32,6 +31,7 @@ def __new__(mcls, name, *args, **kwargs):
             )
 
         if not hasattr(cls, 'group_name'):
+            from .helpers import camel_to_snake
```

Comment:
I've moved this, since the import on the module level caused a circular import. I'm not 100% sure if this could cause problems or would be bad practice. Same goes for the change in `permissions.py` @cobalamin could reassure this change?

Reply:
It's okay, but it's a bit weird. I think it's better to resolve this the other way around - import Group/Permission in the `create_`  commands. It seems more sensible to me to do the non-top-level import for creation commands.